integration/docker-compose.xpu.yml

```diff
@@ -50,6 +50,7 @@ services:
       context: xpu-cpu
       dockerfile: Dockerfile.telegraf
     network_mode: service:xpu-cpu-ssh
+    privileged: true
```

Comment:
I guess we need it to monitor actually CPU,MEM and other really resources outside of container

Reply:
Yes.  That said, I suspect that we don't actually need _all_ privileges and could find the proper subset needed.